commit 06f7054bea1bf6767c0bf48b488fcca2fcfc851e
Author: Julien Rouhaud <julien.rouhaud@free.fr>
Date:   Tue Oct 26 09:13:26 2021 +0800

    Fix compatibility with python 3.9.
    
    The deprecated threading.isAlive() was removed in python 3.9, so use the
    expected threading.is_alive() spelling instead.
    
    This spelling was introduced in python 2.6, which looks old enough as a
    target supported version.
    
    Thanks to Christoph Berg for the report.

--- a/powa_collector/__init__.py
+++ b/powa_collector/__init__.py
@@ -214,7 +214,7 @@ class PowaCollector():
             self.logger.info('List of workers:')
 
         for k, worker in self.workers.items():
-            # self.logger.info(" %s%s" % (k, "" if (worker.isAlive()) else
+            # self.logger.info(" %s%s" % (k, "" if (worker.is_alive()) else
             #                             " (stopped)"))
             worker_srvid = self.config["servers"][k]["srvid"]
 
@@ -225,7 +225,7 @@ class PowaCollector():
             status = "Unknown"
             if (worker.is_stopping()):
                 status = "stopping"
-            elif (worker.isAlive()):
+            elif (worker.is_alive()):
                 status = worker.get_status()
             else:
                 status = "stopped"
@@ -245,7 +245,7 @@ class PowaCollector():
 
         # check for removed servers
         for k, worker in self.workers.items():
-            if (worker.isAlive()):
+            if (worker.is_alive()):
                 continue
 
             if (worker.is_stopping()):
@@ -257,7 +257,7 @@ class PowaCollector():
 
         # check for added servers
         for k in config_new["servers"]:
-            if (k not in self.workers or not self.workers[k].isAlive()):
+            if (k not in self.workers or not self.workers[k].is_alive()):
                 self.logger.info("%s has been added, registering it..." % k)
                 self.register_worker(k, config_new["repository"],
                                      config_new["servers"][k])
